Taskfile.yaml

```diff
@@ -437,7 +437,7 @@ tasks:
     desc: "Tag the featurebyte-server docker image with the version extracted from running 'featurebyte version'."
     vars:
       FB_VERSION:
-        sh: poetry run featurebyte version | cut -d' ' -f2
+        sh: cat pyproject.toml | grep -oP '(?<=^version = ")[^"]+'
```

Comment:
For those running notebook test locally, this grep expression is only available on GNU grep. macos grep would not work with this query